CHANGELOG.md

```diff
@@ -8,6 +8,115 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+| [Manufacturing as a Servie (MaaS) KIT](https://eclipse-tractusx.github.io/docs-kits/category/manufacturing-as-a-service-kit) |                                                                           n/a                                                                           |                                                                                                                             [1.0.0](https://eclipse-tractusx.github.io/docs-kits/kits/Manufacturing%20as%20a%20Service%20Kit/Manufacturing%20as%20a%20Service%20KIT%20Changelog)                                                                                                                             |
+| [Modular Production KIT](https://eclipse-tractusx.github.io/docs-kits/category/modular-production-kit)                       |                                                                           n/a                                                                           |                                                                                                                                         [1.0.0](https://eclipse-tractusx.github.io/docs-kits/kits/Modular%20Production%20Kit/Modular%20Production%20Kit%20Changelog)                                                                                                                                         |
+| [Policy Hub](https://github.com/eclipse-tractusx/policy-hub)                                                                 |                            policy-hub: [1.0.0](https://github.com/eclipse-tractusx/policy-hub/releases/tag/policy-hub-1.0.0)                            |                                                                                                                                                            [1.0.0](https://github.com/eclipse-tractusx/policy-hub/releases/tag/policy-hub-1.0.0)                                                                                                                                                             |
+| [Portal](https://github.com/eclipse-tractusx/portal)                                                                         |                                  portal: [1.8.1](https://github.com/eclipse-tractusx/portal/releases/tag/portal-1.8.1)                                  | Registration: [v2.0.0](https://github.com/eclipse-tractusx/portal-frontend-registration/releases/tag/v2.0.0)<br/><br/>Frontend: [v2.0.0](https://github.com/eclipse-tractusx/portal-frontend/releases/tag/v2.0.0)<br/>Backend: [v1.8.1](https://github.com/eclipse-tractusx/portal-backend/releases/tag/v1.8.1)<br/> Assets: [v1.8.0](https://github.com/eclipse-tractusx/portal-assets/releases/tag/v1.8.0) |
```

Comment:
please change to versions according to https://github.com/eclipse-tractusx/sig-release/issues/671

portal: [2.0.0](https://github.com/eclipse-tractusx/portal/releases/tag/portal-2.0.0)

Registration: [v2.0.0](https://github.com/eclipse-tractusx/portal-frontend-registration/releases/tag/v2.0.0)
Frontend: [v2.0.0](https://github.com/eclipse-tractusx/portal-frontend/releases/tag/v2.0.0)
Backend: [v2.0.0](https://github.com/eclipse-tractusx/portal-backend/releases/tag/v2.0.0)
Assets: [v2.0.0](https://github.com/eclipse-tractusx/portal-assets/releases/tag/v2.0.0)

[Note: not yet released but will be released in time]

Reply:
on it